database/migrations/2020_05_29_102439_create_survey_response2s_table.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateSurveyResponse2sTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('survey_response2s', function (Blueprint $table) {
```

Comment:
I don't think we should hard code these because we will have a hard time if we are told to expand the range of the data we collect. Let us make them dynamic, we can create seeders to add the predefined responses.

Reply:
OK. noted.